### PR TITLE
Cover multi DC environment test case for VCH wizard

### LINKED ISSUE #214: Add tests to ensure provisioning tests include both single datacenter and multiple datacenter environments.
See #208 #209.

We should ensure we add CI tests testing create with VCs that only have one datacenter as well as multiple datacenter. We should also verify paths work for resource pools, clusters, and hosts as well. This issue is to track the adding of those tests.

### PR DESCRIPTION
**Pull request by jooskim**
Fixes #214

### CODE REVIEW

File: tests/manual-test-cases/Group18-VIC-UI/kickoff-tests.robot [hunk 1]
```diff
@@ -38,8 +38,8 @@ Prepare Testbed
     Check Working Dir
     Check Drone
     Check Govc
-    Install VIC Product OVA  6.0u2  ${BUILD_3634791_IP}  10.192.42.90  datastore1 (3)
-    Install VIC Product OVA  6.5d  ${BUILD_5318154_IP}  10.160.75.194  datastore1 (1)
+    Install VIC Product OVA  6.0u2  ${BUILD_3634791_IP}  %{OVA_ESX_IP_VC60U2}  %{OVA_ESX_DATASTORE_VC60U2}
+    Install VIC Product OVA  6.5d  ${BUILD_5318154_IP}  %{OVA_ESX_IP_VC65D}  %{OVA_ESX_DATASTORE_VC65D}
```
Comment (jak-atx) [inline tests/manual-test-cases/Group18-VIC-UI/kickoff-tests.robot]: On the download page for vsphere I see U1d listed as the latest vsphere. https://my.vmware.com/en/web/vmware/info/slug/datacenter_cloud_infrastructure/vmware_vsphere/6_5

Is that the version we are testing? If not when can we switch to testing this version? We may be testing an obsolete version.

File: tests/manual-test-cases/Group1-VCH-Creation-Wizard/1-01-Basic-VCH-Create.robot [hunk 2]
```diff
@@ -54,38 +59,59 @@ Create And Delete VCH On A Single Cluster Environment
...
 
-    # delete extra clusters
-    ${out}=  Run  govc object.destroy /Datacenter/host/Cluster2
-    Should Be Empty  ${out}
-    ${out}=  Run  govc object.destroy /Datacenter/host/Cluster3
-    Should Be Empty  ${out}
-
     # report pass/fail
-    Should Be Equal As Integers  ${rc}  0
+    Should Be Equal As Integers  ${rc}  0
```
Comment (jak-atx) [inline tests/manual-test-cases/Group1-VCH-Creation-Wizard/1-01-Basic-VCH-Create.robot]: add a return so it doesn't complain

File: tests/manual-test-cases/Group18-VIC-UI/kickoff-tests.robot (hunk 1 above)
Comment (jooskim) [reply tests/manual-test-cases/Group18-VIC-UI/kickoff-tests.robot]: No, we are testing against 6.5d not 6.5u1d. The plugin works on 6.5u1 but haven't yet been tested on 6.5u1d. I plan to set up a vCenter 6.5u1 (or 6.5u1d) instance soon on our Dell server and replace the obsolete 6.5d but doing so itself is a major work that might take more effort and time than you think. If I go ahead and do it I expect at least one day of down time for our nightly tests

**Comment by mdubya66** [comment]
Adding this as a proposed 1.3.1 addition given the escapes we saw in 1.3.0 around these areas. @pdaigle

**Pull request merged**